cz.zeropage.Formiko.json

```diff
@@ -1,19 +1,23 @@
 {
     "app-id": "cz.zeropage.Formiko",
     "runtime": "org.gnome.Platform",
-    "runtime-version": "40",
+    "runtime-version": "42",
     "sdk": "org.gnome.Sdk",
     "command": "formiko",
     "rename-icon": "formiko",
     "rename-desktop-file": "formiko.desktop",
     "rename-appdata-file": "formiko.metainfo.xml",
     "finish-args" : [
         "--share=ipc",
-        "--socket=x11",
+        "--device=dri",
+        "--socket=fallback-x11",
         "--socket=wayland",
         "--share=network",
         "--filesystem=home",
         "--filesystem=/media",
+        "--filesystem=xdg-run/gvfs",
+        "--filesystem=xdg-run/gvfsd",
+        "--talk-name=org.gtk.vfs",
```

Comment:
this line should never have been added.

Reply:
https://github.com/flathub/cz.zeropage.Formiko/pull/4